docs/unlocked_package.rst

```diff
@@ -1,14 +1,74 @@
+name and namespace specified in the task options (defaulting to the name and namespace from the 
+``project__package`` section of ``cumulusci.yml``). If a matching package doesn't exist yet, it will be created.
+
+The task then submits a request to create the package version, and once completed (which can take some time), 
+the task will output some information including the SubscriberPackageVersion Id, which can be used to install the package in another org.
+
+If a package version already exists with the exact same contents, its Id will be returned instead of creating a new package version.
+
 
-Promote a package version
+
+Handling Dependencies
+---------------------
+If your project has dependencies configured in the ``project`` section of ``cumulusci.yml``, 
+CumulusCI will try to convert them into a Subscriber Package Version Id (``04t`` key prefix), 
+which is the format required for dependencies in the API for creating a package version.
+
+For dependencies that are specified as a managed package namespace and version, 
+or dependencies specified as a GitHub repository with releases that can be resolved to a namespace and version, 
+CumulusCI needs a scratch org with the dependencies installed in order to do this conversion.
+This is the purpose of the scratch org passed to the ``create_package_version`` task (``dev`` in the example above).
+You must make sure the dependencies are installed in this org before running the 
+``create_package_version`` task. The scratch org definition file from this scratch org 
+will also be used to specify the correct org shape when building the new package version.
+
+For dependencies that are an unpackaged bundle of metadata, CumulusCI will create an additional unlocked package to contain them.
+
+
+
+Promote a Package Version
 -------------------------
+In order to be installed in a production org, an unlocked package version must be
+`promoted <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_create_pkg_ver_promote.htm>`_
+to mark it as released.
 
-Install the unlocked package
+CumulusCI does not yet provide any tools to help with this, so for now you must use the ``sfdx force:package:version:promote`` command.
+If additional unlocked packages were created to hold unpackaged dependencies, they must be promoted as well.
+
+
+
+Install the Unlocked Package
 ----------------------------
+There are two methods available for installing a package in an org: 
+
+* `Install via the Salesforce CLI <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_install_pkg_cli.htm>`_
+* `Install via an installation URL <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_install_pkg_ui.htm>`_
+
+.. note:: Packages can only be installed in Scratch orgs, Sandbox orgs, DE orgs, and Production orgs.
```

Comment:
I'm not sure what this comment is meant to exclude - could we be more explicit?

Reply:
I think I included this because I saw it in the dev docs somewhere. But now that you bring it up, it does look like it really isn't excluding anything. I'll go ahead and remove it.